src/custom-element.js

```diff
@@ -783,6 +782,10 @@ function createBaseAmpElementProto(win) {
    * @final @this {!Element}
    */
   ElementProto.attachedCallback = function() {
+    this.classList.add('-amp-element');
```

Comment:
Done. I chose (1) because `build` should occur strictly after `attach` according to the lifecycle documentation in `base-element.js`.